src/benchmarklib/column_generator.hpp

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <memory>
+#include <random>
+#include <set>
+#include <utility>
+#include <vector>
+
+#include "storage/table_column_definition.hpp"
+
+#include "table_generator.hpp"
+
+namespace opossum {
+
+class Table;
+
+class ColumnGenerator {
```

Comment:
A lot of code. Can you give a reason to include all this code?

Reply:
A question for Thursday.

Reply:
The column generator is used in our benchmarks (hash_join_benchmark.cpp). We discussed with @klauck if we should keep or delete them. Since the benchmarks are to be deleted, this column generator is no longer needed.